ivy_tests/test_ivy/test_frontends/test_torch/test_non_linear_activation_functions.py

```diff
@@ -895,6 +895,41 @@ def test_torch_leaky_relu_(
     )
 
 
+
+# hardswish
+@handle_cmd_line_args
+@given(
+    dtype_and_input=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        safety_factor_scale="log",
+    ),
+    num_positional_args=helpers.num_positional_args(
```

Comment:
Yea we can just hardcode this as discussed :)

Reply:
Yeah I am changing straight away